src/test/java/bio/terra/service/tabulardata/google/BigQueryPdaoUnitTest.java

```diff
@@ -986,6 +994,78 @@ public void testCreateSnapshotWithProvidedIdsWithInvalidRowIds() throws Interrup
                 + ") AS T");
   }
 
+  @Test
+  public void aggregateSnapshotTableTest() {
+    String stringTest = "hello";
+    int intTest = 1234567;
+    List<String> listTest = List.of("a", "b", "c");
+
+    List<FieldValueList> listOfFieldValueList =
+        List.of(
+            FieldValueList.of(
+                List.of(
+                    FieldValue.of(FieldValue.Attribute.PRIMITIVE, stringTest),
+                    FieldValue.of(FieldValue.Attribute.PRIMITIVE, intTest),
+                    FieldValue.of(
+                        FieldValue.Attribute.REPEATED,
+                        listTest.stream()
+                            .map(s -> FieldValue.of(FieldValue.Attribute.PRIMITIVE, s))
+                            .collect(Collectors.toList())))));
+
+    List<StandardSQLTypeName> standardSQLTypeNames =
+        List.of(StandardSQLTypeName.STRING, StandardSQLTypeName.INT64, StandardSQLTypeName.ARRAY);
+
+    Schema schema =
+        Schema.of(
+            FieldList.of(
+                standardSQLTypeNames.stream()
+                    .map(
+                        s -> {
+                          Field.Builder fieldBuilder = Field.newBuilder(s.name(), s);
+                          if (s == StandardSQLTypeName.ARRAY) {
+                            fieldBuilder.setType(StandardSQLTypeName.STRING);
+                          }
+                          return fieldBuilder.build();
+                        })
+                    .collect(Collectors.toList())));
+
+    Page<FieldValueList> page =
```

Comment:
You could mock this instead, it'd be a little shorter.

Reply:
I ended up following the other tests in this file, which is to create a method named `mockX` that returns an `X`.